sdk/typespec/typespec_macros/README.md

```diff
@@ -0,0 +1,81 @@
+# TypeSpec Macros crate for Rust
+
+The TypeSpec Macros crate provides procedural macros for [TypeSpec](https://typespec.io)-generated client libraries. These macros simplify the implementation of common patterns required when working with TypeSpec-generated code.
+
+[Source code] | [Package (crates.io)] | [API reference documentation] | [TypeSpec documentation]
+
+## Getting started
+
+> **Note:** This crate is primarily intended for internal use by the `typespec_client_core` crate. Direct usage is discouraged unless you are contributing to or debugging the TypeSpec ecosystem.
+### Install the package
```

Comment:
Install the Microsoft Learn pack so you get markdown lint and other required checks for public docs (at least on learn.microsoft.com, but many of us try to maintain the same standards here). I do for all the other READMEs and markdown docs herein.

Reply:
I have the learn pack installed. It didn't squiggle on this.